app/assets/javascripts/to_do_items.coffee

```diff
@@ -4,7 +4,7 @@
 
 class @TodoItem
   initialize: () ->
-    bindAddEvents()
+    bindAddEvents this
```

Comment:
Hey @vlasiak, no need to pass `this` as parameter. 2 commnets:
- It is simply accessible inside the method in the same way using `this`.
- Use [fat arrow](http://coffeescript.org/#fat-arrow) CoffeeScript feature for binding. 

FYI: take a look [at usage](http://coffeescript.org/#operators) of `@` for `this` in CoffeeScript.